{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -67,7 +67,7 @@
     )
     with open(full_path, encoding="utf-8") as init_task_file:
         init_task: str = init_task_file.read()
-    hooks.Filters.CLI_DO_INIT_TASKS.add_item((service, init_task))
+        hooks.Filters.CLI_DO_INIT_TASKS.add_item((service, init_task))
```

Comment:
The indent line is not incorrect here. The with block is setting the value for the variable. The variable will be available outside the context manager. You can see various plugins in overhangio repo that have this same code.